sandbox-web/src/main/java/com/exadel/sandbox/team5/DiscountRestController.java

```diff
@@ -25,6 +26,19 @@ public Discount getDiscount(@PathVariable Long id) {
         return service.getAll();
     }
 
+    //Возвращаю при пустом поле поиска или отсутсвие заданых критериев пока пустой список
+    @GetMapping("/search")
+    public List<Discount> getBySearchWord(@RequestParam String searchWord) {
```

Comment:
Please use another name of method. We can use not only one word but some sort of phrase, for example:
```
search(String searchText)
```

Reply:
Done